cart/interfaces/graphql/mutationresolvers.go

```diff
@@ -252,6 +252,34 @@ func (r *CommerceCartMutationResolver) CartClean(ctx context.Context) (bool, err
 	return true, nil
 }
 
+// UpdateAdditionalData of cart
+func (r *CommerceCartMutationResolver) UpdateAdditionalData(ctx context.Context, additionalDataList []*dto.KeyValue) (*dto.DecoratedCart, error) {
+	session := web.SessionFromContext(ctx)
+	for _, additionalData := range additionalDataList {
+		_, err := r.cartService.UpdateAdditionalData(ctx, session, additionalData.Key, additionalData.Value)
```

Comment:
This will lead to a lot of backend calls, e.g. 10 key:value pairs will lead to 10 calls. See `CartService` comments to have a function to update all additional data at once.